src/internal_events/remap.rs

```diff
@@ -6,19 +6,22 @@ pub struct RemapEventProcessed;
 
 impl InternalEvent for RemapEventProcessed {
     fn emit_metrics(&self) {
-        counter!("processed_events_total", 1);
+        counter!("processed_events_total", 1,
+                 "component_kind" => "transform",
+                 "component_type" => "remap",
```

Comment:
@MOZGIII shouldn't these be unnecessary with the tokio trace context?

Reply:
Correct, these are no longer needed